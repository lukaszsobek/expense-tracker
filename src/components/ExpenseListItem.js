import React from "react"
import { Link } from "react-router-dom"

export const ExpenseListItem = ({
        id, description, history, amount, createdAt
    }) => {
    return (
        <div>
            <div>
            <Link to={`/edit/${id}`}>{description}</Link>
                - {amount}
                - {createdAt}
            </div>
        </div>
    )
}

export default ExpenseListItem