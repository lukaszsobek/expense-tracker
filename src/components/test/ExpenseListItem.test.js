import React from "react"
import { shallow } from "enzyme"

import { ExpenseListItem } from "../ExpenseListItem"

test("Renders ExpenseListItem", () => {
        const props = {
            id: "1", description: "2", history: "3", amount: "4", createdAt: 0
        }
        const wrapper = shallow(<ExpenseListItem {...props} />)
        expect(wrapper).toMatchSnapshot()
})