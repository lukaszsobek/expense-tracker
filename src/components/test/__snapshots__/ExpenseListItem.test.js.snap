// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ExpenseListItem 1`] = `
<div>
  <div>
    <Link
      replace={false}
      to="/edit/1"
    >
      2
    </Link>
    - 
    4
    - 
    5
  </div>
</div>
`;
